src/tasks/block.rs

```diff
@@ -174,7 +176,7 @@ impl BlockBuilder {
         match bundles {
             Ok(bundles) => {
                 for bundle in bundles {
-                    let result = self.simulate_bundle(&bundle.bundle, host_provider).await;
+                    let result = self.simulate_bundle(&bundle.bundle, ru_provider).await;
```

Comment:
what are we simulating the bundle against? we don't have a simulation engine yet, nor we have a node we can use for this.